go.mod

```diff
@@ -48,6 +48,7 @@ require (
 require (
 	contrib.go.opencensus.io/exporter/ocagent v0.7.1-0.20200907061046-05415f1de66d // indirect
 	contrib.go.opencensus.io/exporter/prometheus v0.4.2 // indirect
+	github.com/Azure/agentbaker v0.20240304.0-devinwon // indirect
```

Comment:
Is there code I can review on agentbaker side for when this is prepared

Reply:
Alex has been reviewing the agentbaker side changes but I am happy to have you reviewed moving on. :)